sql.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2014 Google Inc.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package uuid
+
+import (
+	"database/sql"
+	"database/sql/driver"
```

Comment:
Ok fair enough. I'll remove the dep on sql and sql/driver. The sql dep is only there for the little check I added in to make sure it implements the interface at compile time. I'll submit an update with just Scan implemented.